src/Controls/src/Core/HandlerImpl/VisualElement.Impl.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Microsoft.Maui.Layouts;
+
+namespace Microsoft.Maui.Controls
+{
+	public partial class VisualElement : IFrameworkElement
+	{
+		private IViewHandler _handler;
```

Comment:
Move IFrameworkElement up to VisualElement so that Page also implements IFrameworkElement

Reply:
I like this because if we ever do shapes or some non-view control, we can still lay it out.